man/man1/p11sak.1.in

```diff
@@ -432,3 +437,40 @@ prints help for the usage of
 .B p11sak
 and/or the respective command.
 .PP
+.
+.
+.
+.SH "p11sak_defined_attrs.conf"
```

Comment:
The section head would be "FILES".  See `man man-pages` for the usual section heads.

Reply:
Done